### ISSUE TITLE
Set-Output is deprecated.

### ISSUE THREAD

**Issue by djthornton1212** [mathze/workflow-dispatch-action#56]
**Describe the bug**
```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**To Reproduce**
1. Add action to workflow
2. Run workflow

**Expected behavior**
Utilization of late actions/core, thus no warnings.

**Data/Screenshots**
```
Run mathze/workflow-dispatch-action@v1.1.0
Reading inputs
Going to wait until run 3300587560 completes
No result, retry in 30s
No result, retry in 30s

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/[20](https://github.com/caregility/action-testing/actions/runs/3299978915/jobs/5445234705#step:5:22)22-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**System (please complete the following information):**
- Runner-OS: [ubuntu-latest]
- Action-Version: [v1.1.0]

**Additional context**
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

**Comment by mathze** [comment]
Hi @djthornton1212,

unfortunately, this is out of my hands as it came from the used dependency.
But I will try to contact the library-owner to get an update.

**Comment by mathze** [comment]
Fixed with #69
